plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -1252,4 +1254,16 @@ public HiveConfig setPartitionProjectionEnabled(boolean enabledAthenaPartitionPr
         this.partitionProjectionEnabled = enabledAthenaPartitionProjection;
         return this;
     }
+
+    public S3StorageClassFilter getS3StorageClassFilter()
+    {
+        return s3StorageClassFilter;
+    }
+
+    @Config("hive.s3.storage-class-filter")
```

Comment:
Please document this field in `legacy-s3.md` file.
Also add `@ConfigDescription`

Reply:
Apologies for the back and forth, this setting is not related to legacy or native s3 client, but rather to hive connector itself.

Please restore the documentation into `hive.md`.
I'd say to add it under https://github.com/trinodb/trino/blob/f992602dd43a81b72dc6044d37af2fb006e38d28/docs/src/main/sphinx/connector/hive.md?plain=1#L84